### PR TITLE
Build deb packages for ubuntu 16.04

### PR DESCRIPTION
**Pull request by tmatsuo**
I'm copying over to the new directory. See the 2nd commit for the actual changes.

### CODE REVIEW

File: ubuntu-package-builder/Dockerfile [hunk 1]
```diff
@@ -73,8 +102,16 @@ RUN echo "deb http://ftp.debian.org/debian jessie-backports main" > /etc/apt/sou
         automake \
         autotools-dev
 
-COPY build.sh /
 RUN mkdir -p /workspace
 WORKDIR /workspace
 
-ENTRYPOINT ["/build.sh"]
+COPY debian /workspace/debian
```
Comment (chingor13) [inline ubuntu-package-builder/Dockerfile]: Isn't `/workspace` mounted over during the cloud build process?

File: ubuntu-package-builder/libraries/libvips/build.sh [hunk 2]
```diff
@@ -28,4 +28,4 @@ if [ ! -f "${ARTIFACT_LIB_DIR}/${OUTPUT_FILE}" ]; then
     dpkg-buildpackage -us -uc -j"$(nproc)"
     cp ../*.deb ${ARTIFACT_LIB_DIR}
     popd
-fi
+    fi
```
Comment (chingor13) [inline ubuntu-package-builder/libraries/libvips/build.sh]: nit: extra whitespace

File: ubuntu-package-builder/Dockerfile (hunk 1 above)
Comment (tmatsuo) [reply ubuntu-package-builder/Dockerfile]: right, removed in the next cl

File: ubuntu-package-builder/libraries/libvips/build.sh (hunk 2 above)
Comment (tmatsuo) [reply ubuntu-package-builder/libraries/libvips/build.sh]: done

**Pull request merged**